src/main/resources/native/android/c/launcher.c

```diff
@@ -24,20 +25,36 @@ extern int to_jfx_touch_action(int state);
 
 ANativeWindow *window;
 jfloat density;
+char* appDataDir;
 
 int start_logger(const char *app_name);
 static int pfd[2];
 static pthread_t thr;
 static const char *tag = "myapp";
-const char * args[] = {
+const char * origargs[] = {
         "myapp",
         "-Djavafx.platform=android",
         "-Dembedded=monocle",
         "-Dglass.platform=Monocle",
         "-Djavafx.verbose=true",
-        "-Djavafx.pulseLogger=true",
         "-Dprism.verbose=true"};
 
+int argsize = 6;
+
+char** createArgs() {
+    int origSize = sizeof(origargs)/sizeof(char*);
+    char** result = malloc((origSize+1)* sizeof(char*));
+    for (int i = 0; i < origSize; i++) {
+        result[i] = origargs[i];
+    }
+    int tmpArgSize=18+strlen(appDataDir);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4782774)